### PR TITLE
Selenium: fix unstable selenium tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR:
- add checking that java language server if fully initialized before working with a project in ```CheckOnValidAndInvalidPackageNameTest``` and ```CreateWorkspaceFromCamelStackTest``` tests;
- add timeout for 1 second after every clicking on change RAM button in ```NewWorspace``` page object to avoid fast clicking on buttons;
- fix expected locator on "Che Documentation" page ```DocumentationPage``` page object;
- fix expected menu item name for starting a new terminal in ```TheiaBuildPluginTest``` test;
- wait that expected file is visible in Project Explorer in ```RunToCursorTest``` test.

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java [hunk 1]
```diff
@@ -274,6 +274,9 @@ public boolean isMachineExists(String machineName) {
 
   public void clickOnIncrementMemoryButton(String machineName) {
     seleniumWebDriverHelper.waitAndClick(By.xpath(format(INCREMENT_MEMORY_BUTTON, machineName)));
+
+    // we need to wait a little to avoid quick clicking on this button
+    WaitUtils.sleepQuietly(1);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java]: But we have already clicked the button in the command before, haven't it?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java [hunk 2]
```diff
@@ -300,6 +303,9 @@ public void waitRamValueInSpecifiedRange(
 
   public void clickOnDecrementMemoryButton(String machineName) {
     seleniumWebDriverHelper.waitAndClick(By.xpath(format(DECREMENT_MEMORY_BUTTON, machineName)));
+
+    // we need to wait a little to avoid quick clicking on this button
+    WaitUtils.sleepQuietly(1);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java]: But we have already clicked the button in the command before, haven't it?

**Review by dmytro-ndp — APPROVED** [review]

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1284/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1284//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12156
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**